R/RtGam.R

```diff
@@ -83,8 +95,21 @@ RtGam <- function(cases,
       ...
     )
   )
+  diagnostics <- calculate_diagnostics(fit)
+  if (warn_for_diagnostic_failure) {
+    issue_diagnostic_warnings(diagnostics)
+  }
 
-  invisible(NULL)
+  format_for_return(
```

Comment:
is this really a class constructor? If yes what about calling it that (i.e new_RtGam) just for R standard following

Reply:
I don't think it is the R standard for model fitting functions? e..g., `lm()`, `glm()`, `gam()` all return an eponymously classed object. They're class constructors but the standard for unifunctional packages is to just construct with a function of the same name. 

I could have sworn Hadley says something along these lines in R packages 2e but I couldn't find it with a quick cmd+F

Reply:
https://github.com/CDCgov/cfa-gam-rt/pull/22/commits/c5f808cdfab3b01b234351038cd70f216e4e63e3